pkg/kp/flags/kingpin.go

```diff
@@ -5,10 +5,11 @@ package flags
 import (
 	"io/ioutil"
 	"log"
+	"net"
 	"net/http"
 
 	"github.com/square/p2/pkg/kp"
-	"github.com/square/p2/pkg/util/net"
+	netutil "github.com/square/p2/pkg/util/net"
```

Comment:
I could swear a different alias for this package is in use elsewhere. Could you look for that alias and make this consistent?